curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-pyramid-generator/660f280dda5040f707c76b4a.md

```diff
@@ -0,0 +1,46 @@
+---
+id: 660f280dda5040f707c76b4a
+title: Step 35
+challengeType: 0
+dashedName: step-35
+---
+
+# --description--
+
+Now you have a bunch of `#` symbols, but still no pyramid. Thankfully, your `i` variable represents the number of the "row" your loop is currently on, and you can use that to create a pyramid-like structure.
+
+To do this, you will use the `.repeat()` method that strings have. The `.repeat()` method takes a number, which indicates how many times you would like to repeat the given string.
```

Comment:
we should add an example for the repeat method

````suggestion
To do this, you will use the `.repeat()` method that strings have. The `.repeat()` method takes a number, which indicates how many times you would like to repeat the given string.

```js
const activity = "Code! ";

// Code! Code! Code! 
activity.repeat(3);
```
````